alert-common/src/main/java/com/synopsys/integration/alert/common/persistence/accessor/ConfigurationAccessor.java

```diff
@@ -7,26 +7,25 @@
  */
 package com.synopsys.integration.alert.common.persistence.accessor;
 
-import java.util.List;
 import java.util.Optional;
-
-import org.springframework.data.domain.Page;
-import org.springframework.data.domain.Pageable;
+import java.util.UUID;
 
 import com.synopsys.integration.alert.api.common.model.AlertSerializableModel;
 import com.synopsys.integration.alert.api.common.model.exception.AlertConfigurationException;
-import com.synopsys.integration.alert.common.persistence.model.DatabaseModelWrapper;
 import com.synopsys.integration.alert.common.rest.model.AlertPagedModel;
 
 public interface ConfigurationAccessor<T extends AlertSerializableModel> {
-    Optional<DatabaseModelWrapper<T>> getConfiguration(Long id);
+    int FIRST_PAGE_NUMBER = 0;
+    int DEFAULT_PAGE_SIZE = 10;
```

Comment:
I believe these defaults are available in AlertPagedModel

Reply:
Thanks.  I found something I want to switch up.  AlertPagedModel shouldn't depend on the read controller value.  The read controller should depend on the AlertPagedModel value.  I will update this.